src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -410,7 +426,7 @@ export default {
 			// again another time
 			this.$refs.message.classList.remove('highlight-animation')
 		},
-		handleReply() {
+		handleReply(t) {
```

Comment:
```suggestion
		handleReply(token) {
```

t is for translations, please don't overwrite it